tests/DCPS/Compiler/typecode/C++11/.gitignore

```
@@ -0,0 +1,14 @@
+Compiler_Typecode_C++11
+typecodeC.cpp
+typecodeC.h
+typecodeC.inl
+typecodeS.cpp
+typecodeS.h
```

```suggestion
typecodeC.h
```

I don't think these files get generated for the C++11 mapping.